README.md

```diff
@@ -1,83 +1,99 @@
-# Branch-iOS-Invite-SDK
+# Branch-iOS-Invite-SDK User Guide
 
-The purpose of this SDK is to provide an out-of-the-box functional 'invite feature' for apps consuming the Branch SDK that want to utilize a standard invite feature in their app.
+##Introduction
 
-There will still be some configuration on the dashboard, but the goal is to provide the most extensible, yet simple to use, full invite feature SDK. To see the basics of setting up an app with Branch, check out [the Branch iOS SDK readme](https://github.com/BranchMetrics/Branch-iOS-SDK).
+This iOS Invite SDK provides an out-of-the-box functional 'invite feature' for apps consuming the Branch SDK that want to utilize a standard invite feature in their app.
 
-# Invite Flow
-In your app, there will be a trigger to open the Invite UI. This will show a list of contacts and allow the user to select friends they want to invite to join them.  
+This iOS Invite SDK user guide provides you with step-by-step instructions on how to set up the invite feature. If you haven't set up an app with Branch yet, check out [the Branch iOS SDK readme](https://github.com/BranchMetrics/Branch-iOS-SDK).
 
-![Invite](https://s3-us-west-1.amazonaws.com/branchhost/invite_sdk_1.gif)
+##Table of Contents
+The iOS Invite SDK user guide includes the following sections:
 
-# New User Experience (Install/Open & Personal Welcome)
-Invited users will receive a message (via SMS, Email, or a custom provider you've implemented). When they open this URL, they'll be fingerprinted by the Branch server as an "invited user," which will be remember when your app is launched on their device. When Invited Users enter the app, they'll be shown a Welcome UI (either the Branch default, or a custom screen). This Welcome Screen contains the inviting user's image, name, and id. The invited user can choose to accept the invite, or cancel and continue on their own.
+Section Titles  | 
+------------- | 
+[Introduction] (#introduction)| 
+|[Pre-requistes] (#1-pre-requistes)      
+[Installing the iOS Invite SDK](#2-installing-the-iOS-invite-sdk)| 
+[Getting Your Branch Key](#4-getting-your-branch-key)|
+[Adding Your Branch Key to Your Project](#5-adding-your-branch-key-to-your-project)|
+[Showing the Invite Controller and Customizations](#5-showing-the-invite-controller-and-customizations)|
+[Customization](#6-customizations-and-overrides)|
+[Referral Management](#7-referral-management)|
+[Example Usage](#9-example-usage)|
 
-![Open](https://s3-us-west-1.amazonaws.com/branch-guides/click_open.gif)
+##Pre-requistes
+You must have the following completed before you can use the invite feature:
 
-# Usage Guide
+- An account to access the Dashboard and store all of your link data 
+- A configured dashboard for your iOS app
```

Comment:
"A configured app within the Branch dashboard" would be better here. The Dashboard != an app, you can have multiple.